portality/templates/application_form/_value.html

```diff
@@ -0,0 +1,15 @@
+{% macro value(x, field_name, formulaic_context) %}
+    {% if x == None or x == "" or x == [] %}
+        <em>[Nothing]</em>
+    {% elif x is string and x.startswith("http") %}
+        <a href="{{ x }}" target="_blank" rel="noopener" style="word-break: break-all">{{ x }}</a>
+    {% elif x == "y" %}
+        Yes
+    {% elif x == "n" %}
+        No
+    {% elif field_name == "language" or field_name == "publisher_country" or field_name == "institution_country" %}
+        {{ formulaic_context.lang_to_country("EN") }}
```

Comment:
Should `EN` be hard-coded here?

Reply:
It shouldn't, that should be just value converted